cmd/agent/app/check.go

```diff
@@ -271,3 +275,9 @@ func getMetricsData(agg *aggregator.BufferedAggregator) map[string]interface{} {
 
 	return aggData
 }
+func printWindowsUserWarning(op string) {
+	fmt.Printf("\n")
+	color.Yellow("The %s command runs in a different user context than the running service\n", op)
+	color.Yellow("This could affect the results of, if the command relies on specific permissions and/or user context\n")
```

Comment:
It's good to warn the user about this difference that could affect their check. That said, one concern I have is that it's shown as a warning but doesn't give any way to resolve it.
Not sure how best to display this to make it clear that it's not really a warning. Idea: print this with no particular color, and prepend it with `Note: `. Open to other ideas as well.